templates/apps/sumo-aws-apps/sumologic-app-utils/src/awsresource.py

```diff
@@ -343,7 +343,7 @@ def deregister_delegated_administrator(self,account_id: str, service_principal:
             delegated_administrators = self.ORGANIZATIONS_CLIENT.list_delegated_administrators(
                 ServicePrincipal=service_principal)
 
-            logger.debug(str(delegated_administrators))
+            #logger.debug(str(delegated_administrators))
```

Comment:
we can delete such log lines which are not much useful

Reply:
okay